.tp-config.json

```
@@ -6,15 +6,25 @@
   "parameters": {
     "listenerPollTimeout": {
       "description": "Only for Kafka. Timeout to use when polling the consumer.",
+      "default": 3000,
       "required": false
     },
     "listenerConcurrency": {
       "description": "Only for Kafka. Number of threads to run in the listener containers.",
+      "default": 3,
+      "required": false
+    },
+    "asyncapiFileDir": {
+      "description": "Parameter of @asyncapi/generator-hooks#createAsyncapiFile, allows to specify where original AsyncAPI file will be stored.",
+      "default": "src/main/resources/api/",
       "required": false
     }
   },
   "generator": ">=0.46.0 <2.0.0",
```

```suggestion
  "generator": ">=0.47.0 <2.0.0",
```

as otherwise default values will not work